lib/app/Auth/PhoenixOAuthRepository.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Phoenix\Auth;
+
+use DoSomething\Northstar\Contracts\OAuthRepositoryContract;
+use League\OAuth2\Client\Token\AccessToken;
+
+class PhoenixOAuthRepository implements OAuthRepositoryContract {
+  /**
+   * Get the given authenticated user's access token.
+   *
+   * @return \League\OAuth2\Client\Token\AccessToken|null
+   */
+  public function getUserToken() {
+    // @TODO: Not in use for now.
+    return NULL;
+  }
+
+  /**
+   * Get the OAuth client's token.
+   *
+   * @return \League\OAuth2\Client\Token\AccessToken|null
+   */
+  public function getClientToken() {
+    $clientId = variable_get('dosomething_northstar_oauth_client_token_clientid', NULL);
```

Comment:
Yeah, I was referencing the example from the `LaravelOAuthRepository` in the NS-PHP package, and kinda realized that's only really used as an id to find the record in the DB... wasn't sure if we'd _want_/_need_ it here in Phoenix land.